packages/numbers/lib/numbers.ts

```diff
@@ -20,45 +25,93 @@ const runRequest = async <T>(options: NumbersClassParameters): Promise<NumbersRe
     return result;
 }
 
+const remapObjects = <T, O>(mapping, newObject: T, oldObject: O): T => {
+    for (let key in mapping) {
```

Comment:
```suggestion
    for (const key in mapping) {
```
Had to look this up to make sure. haha https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in